index.js

```diff
@@ -137,12 +137,9 @@ module.exports = themeConfig => {
       }
       if (themeConfig.summary) {
         pageCtx.summary =
-          removeMd(
-            strippedContent
-              .trim()
-              .replace(/^#+\s+(.*)/, '')
-              .slice(0, themeConfig.summaryLength)
-          ) + ' ...'
+          removeMd(strippedContent.trim())
+            .replace(/^#+\s+(.*)/, '')
```

Comment:
In this case, title will not be removed.

```suggestion
removeMd(strippedContent.trim() .replace(/^#+\s+(.*)/, ''))
```

Reply:
If the replace is made before removeMd, the content might end up with a broken markdown again. Can you explain why the replace regex is needed?

Reply:
Sorry for the late reply, I didn't notice I haven't submit the pending comment